.github/workflows/tests.yml

```diff
@@ -29,6 +29,7 @@ jobs:
   build:
     needs: wsi-artifacts
     runs-on: ${{ matrix.os }}
+    continue-on-error: true
```

Comment:
is this needed? I don't see why we should continue the CI if we do not have these fixtures

Reply:
yes you're right, leftovers